### ISSUE TITLE
Unified subscription list

### ISSUE THREAD

**Issue by cellio** [codidact/qpixel#285]
https://meta.codidact.com/questions/276419

Depends on: https://github.com/codidact/qpixel/issues/284 (network profile).

> If I navigate to my list of question subscriptions, all I see is my subscriptions for the current community.  Is there a way to combine all of the question subscriptions into one list as well?

**Comment by cellio** [comment]
Marked deferred because of the dependency.  (If there's a more proper way to manage dependencies, someone should please feel free to edit.)
